### PR TITLE
Fixed observation ( in separate branch)

### PR DESCRIPTION
**Pull request by csumano**
fixed observation

### CODE REVIEW

File: lsstbroker/observation.py [hunk 1]
```diff
@@ -4,13 +4,17 @@
 to classify the data based on this information.
 """
 
-class TransientOberservation(object):
-
-    time = None
```
Comment (ExcaliburZero) [inline lsstbroker/observation.py:9]: Keep these here.

File: lsstbroker/observation.py [hunk 2]
```diff
@@ -4,13 +4,17 @@
...
-    light = None
-    color = None
+class Oberservation(object):
 
     def __init__(self, time, light_value, color):
+
+        self.time = None
```
Comment (ExcaliburZero) [inline lsstbroker/observation.py]: Set these to be equal to the corresponding parameters.

File: lsstbroker/observation.py [hunk 3]
```diff
@@ -4,13 +4,17 @@
...
+        self.light = None
+        self.color = None
+
+
+
+
         """
```
Comment (ExcaliburZero) [inline lsstbroker/observation.py:14]: Move this section to right after the `def __init__`.

**Review by ExcaliburZero — CHANGES_REQUESTED** [review]
Looks good, just needs a few changes.

**Comment by csumano** [comment]
alright ! made the changes

File: lsstbroker/observation.py [hunk 4]
```diff
@@ -16,3 +16,7 @@ def __init__(self, time, light_value, color):
         to geneerate a transients time, light-value and color, it will return
         a tuple.
         """
+
+        self.time = time
+        self.light = light
```
Comment (ExcaliburZero) [inline lsstbroker/observation.py]: Change this to `self.light = light_value`

**Review by ExcaliburZero — CHANGES_REQUESTED** [review]
One more change.

**Comment by csumano** [comment]
fixed

File: lsstbroker/observation.py [hunk 5]
```diff
@@ -16,3 +16,7 @@ def __init__(self, time, light_value, color):
         to geneerate a transients time, light-value and color, it will return
         a tuple.
         """
+
+        self.time = time
+        self.light = light_value
+        self.color = color
```
Comment (ExcaliburZero) [inline lsstbroker/observation.py]: Can you change all instances of color to error for now, since that is what the RR Leary data uses?

**Review by ExcaliburZero — CHANGES_REQUESTED** [review]
One more change, with the RR Leary classifier in mind.

**Comment by csumano** [comment]
changed color instance to error

**Review by ExcaliburZero — APPROVED** [review]

**Comment by ExcaliburZero** [comment]
Merged in 98163f25dfe7e63281bf856745381be89dfa5a83

**Pull request closed without merging**